cogeo_mosaic/create.py

```diff
@@ -123,10 +171,10 @@ def create_mosaic(
             continue
 
         intersections_geoms = [dataset_geoms[idx] for idx in intersections_idx]
-        intersections = [results[idx] for idx in intersections_idx]
+        intersections = [features[idx] for idx in intersections_idx]
 
         dataset = [
-            {"path": f["properties"]["path"], "geometry": geom}
+            {"identifier": accessor(f), "geometry": geom}
```

Comment:
I realized the `accessor` should actually get called here currently. In #59 I refactor this a little so that you don't need to create a new `list` of `dict`s.